src/components/dropdown/_dropdown.scss

```diff
@@ -92,7 +92,7 @@
     position: absolute;
     z-index: z('dropdown');
     max-height: 0;
-    transition: max-height $transition--expansion $carbon--ease-out;
+    transition: max-height $carbon--ease-in $transition--expansion $carbon--ease-out;
```

Comment:
Seems that the property value is invalid according to https://developer.mozilla.org/en-US/docs/Web/CSS/transition#Formal_syntax - I no longer see the animation?